app/components/views/HelpPage/LogsTab/Page.jsx

```diff
@@ -26,95 +24,42 @@ const Logs = ({
   onHideDcrlndLogs
 }) => (
   <>
-    {/*
-      TODO Create shared component and use it here and on the privacy page logs.
-    */}
     <Subtitle title={<T id="logs.subtitle" m="System Logs" />} />
-    {!isDaemonRemote ? (
-      !showDcrdLogs ? (
-        <div className={classNames(styles.logArea, styles.hidden)}>
-          <div
-            className={classNames(styles.title, styles.hidden)}
-            onClick={onShowDcrdLogs}>
-            <T id="help.logs.dcrd" m="dcrd" />
-          </div>
-        </div>
-      ) : (
-        <div className={classNames(styles.logArea, styles.expanded)}>
-          <div
-            className={classNames(styles.title, styles.expanded)}
-            onClick={onHideDcrdLogs}>
-            <T id="help.logs.dcrd" m="dcrd" />
-          </div>
-          <div className={styles.logs}>
-            <textarea rows="30" value={dcrdLogs} disabled />
-          </div>
-        </div>
-      )
-    ) : (
-      <div />
+    {!isDaemonRemote && (
```

Comment:
I dont think this is working properly, I am seeing an empty dcrd log even when I am connected to a remote dcrd.

Reply:
:eyes:

Reply:
checking

Reply:
I'm seeing the same on master - will fix that in this PR.

Reply:
fixed 🧘‍♂️